lib/getIsPro.ts

```diff
@@ -0,0 +1,20 @@
+import { Address } from "viem";
+import { API_APP_URL } from "./consts";
+
+async function getIsPro(address: Address, maxRetries = 1) {
```

Comment:
is this lib necessary anymore?
- what are the side effects of removing `getIsPro`?